Core/src/IfcGeometryConverter/SpiralUtils.h

```diff
@@ -205,9 +205,16 @@ namespace OpenInfraPlatform
 							localPolynomialConstantCnt,
 							s
 						);
-
+#ifdef _DEBUG
+					double	rValueNumericalIntegration =
+						NumericalIntegrationSin(
+							localPolynomialConstants,
+							localPolynomialConstantCnt,
+							s
+						);
+#endif // _DEBUG
 					//delete[] localPolynomialConstants;
-
+					if (std::fabs(rValueNumericalIntegration - rValue) > 0.000001) throw oip::InconsistentGeometryException("Incorrect length implementation");
```

Comment:
This statement need to be within ifdef Debug as well, otherwise the release will not compile ...

Reply:
Resolved in [d4d80c8](https://github.com/tumcms/Open-Infra-Platform/pull/553/commits/d4d80c8a2ce0a978560901d0990d8b5543b3332e)